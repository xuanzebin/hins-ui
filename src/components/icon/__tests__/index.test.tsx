import React from 'react'
import { create } from 'react-test-renderer'

import Icon from '../index'

describe('icon', () => {
  it('snapshot test', () => {
    const component = create(<Icon name="clock" />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
