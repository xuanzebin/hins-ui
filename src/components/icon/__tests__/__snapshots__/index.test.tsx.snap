// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon render successfully 1`] = `
<div
  className="icon-wrapper"
>
  <svg
    className="hins-icon"
  >
    <use
      xlinkHref="#clock"
    />
  </svg>
</div>
`;
